django/cantusdb_project/main_app/templates/office_list.html

```diff
@@ -1,6 +1,6 @@
 {% extends "base.html" %}
 {% block content %}
-<title>Office/Mass Abbereviations | Cantus Manuscript Database</title>
+<title>Office/Mass abbereviations | Cantus Manuscript Database</title>
```

Comment:
Nope, this isn't right - we want "Abbreviations", probably including the uppercase initial, and definitely removing the bonus "e" in the middle

Reply:
Ah, I guess I misunderstood the typo in #926, oops!

Reply:
no worries lol - for what it's worth, you got it correct in the PR title :P